Cargo.toml

```diff
@@ -29,13 +29,13 @@ primitive-types = "0.11.1"
 rand = "0.8.5"
 reqwest = "0.11.11"
 ring = "0.16.20"
-rsa = "0.6.1"
+rsa = { version = "0.6.1", optional = true }
 secp256k1 = { version = "0.22.1", optional = true, features = [ "recovery" ] }
 serde = "1.0.132"
 serde_json = "1.0.73"
 sha2 = "0.10.2"
 thiserror = "1.0.30"
-tokio = { version = "1.15.0", features = [ "fs" ]}
+tokio = { version = "1.14.0", features = [ "fs" ]}
```

Comment:
This dependency downgrade is really necessary?

Reply:
Oh, I forgot to mention that in the PR description. Unfortunately it is, for Sugar, because the `solana-client` library pins its tokio dependency to 1.14.1 (I'm not sure why):

```
error: failed to select a version for `tokio`.
    ... required by package `bundlr-sdk v0.2.0 (/Users/samuel/Metaplex/rust-sdk)`
    ... which satisfies path dependency `bundlr-sdk` (locked to 0.2.0) of package `sugar-cli v0.6.1 (/Users/samuel/Metaplex/sugar)`
versions that meet the requirements `^1.15.0` are: 1.20.1, 1.20.0, 1.19.2, 1.19.1, 1.19.0, 1.18.2, 1.18.1, 1.18.0, 1.17.0, 1.16.1, 1.15.0

all possible versions conflict with previously selected packages.

  previously selected package `tokio v1.14.1`
    ... which satisfies dependency `tokio = "~1.14.1"` of package `solana-client v1.10.29`
    ... which satisfies dependency `solana-client = "~1.10.29"` of package `anchor-client v0.25.0`
    ... which satisfies dependency `anchor-client = "^0.25.0"` of package `sugar-cli v0.6.1 (/Users/samuel/Metaplex/sugar)`

failed to select a version for `tokio` which could resolve this conflict
```

However, setting it as "1.14.0" [is the same as](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) "^1.14.0" so it will go up to 1.15 or higher, if can and needs, to for programs relying on Bundlr that need a higher version, so I don't think it's a breaking change.